conf/application.conf

```diff
@@ -201,15 +201,41 @@ microservice {
             port=9998
             originatorId="local"
         }
+
         rti-hod {
             host=localhost
             port=9998
             authorizationToken = "Local"
             env = "local"
             originatorId="local"
         }
-
     }
 }
 
+Dev {
```

Comment:
This service doesn't use quite the same app configuraton pattern as some of our other older services. 
Instead of providing the Dev, Test, and Prod configurations all in this file, we can just provide the default local configuration once. 
Anything that differs from that default local configuration based on environment can then be overriden in the app-config-* configuration repositories.